src/index.css

```diff
@@ -118,3 +119,13 @@ body {
   background-color: #b30000;
   color: white;
 }
+
+.currency-wrapper {
```

Comment:
I don't like the position absolute, while it works, that's not how you align the items. There is no reason for the currency wrapper to be positioned absolute in your display. If you want to fix this, use flex on the sidebar:

.sidebar {
    display: flex;
    align-items: center;
}